### PR TITLE
feat(staff page)-switch layout with ccip

### PR DESCRIPTION
**Pull request by SivanYeh**
## Description
Add a custom layout "ccip" to hide footer and header at page staff, switch the layout based on CCIP value

## Steps to Test This Pull Request
1. Type “<localhost>/2023/en-us/about/staff?ccip=1” in url
2. Scroll to the top see if header is hidden
3. Scroll down the bottom see if footer is hidden

## Expected behavior

When ccip is true, footer and header are hidden as picture:
image

If false:
image

## Related Issue
#413

### CODE REVIEW

**Comment by SivanYeh** [comment]
Gotta figure out how to switch the layout with token, and which CCIP token meant to be the key.
Almost there!

**Review by josix — CHANGES_REQUESTED** [review]

File: layouts/ccip.vue [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        const conferenceName = this.$store.state.configs.conferenceName
+        const conferenceYear = this.$store.state.configs.conferenceYear
+        return {
+            titleTemplate: (titleChunk) => {
+                const title = `${conferenceName} ${conferenceYear}`
+                return titleChunk && titleChunk !== title
+                    ? `${titleChunk} | ${title}`
+                    : title
+            },
+            meta: [
```
Comment (mattwang44) [inline layouts/ccip.vue:27]: `meta` & `script` 內的設定大多是為了有更好的 SEO，但這個特製的 layout 在我想像中不需要這些設定、甚至不希望只會使用這個 layout 的頁面會在搜尋引擎上被搜尋到

File: layouts/ccip.vue [hunk 2]
```diff
@@ -0,0 +1,161 @@
+<template>
+    <div class="default-layout">
+        <div class="default-layout__body">
+            <Nuxt keep-alive :keep-alive-props="{ include: includeArr }" />
```
Comment (mattwang44) [inline layouts/ccip.vue:4]: keep alive 是為了時間表頁面所做的特殊設定 (see #299)，CCIP 頁面不會需要

File: pages/about/staff.vue [hunk 3]
```diff
@@ -284,6 +291,7 @@ export default {
                     { name: 'JunWei Song', photo: 'program/JunWei Song.jpeg' },
                 ],
             },
+            isAppMode: false,
```
[the remark this answers is not in the document]
Comment (SivanYeh) [reply pages/about/staff.vue:294]: Oh mine. It was a deprecated parameter. I will delete it!

File: layouts/ccip.vue (hunk 1 above)
Comment (SivanYeh) [reply layouts/ccip.vue]: Deleted!

File: layouts/ccip.vue (hunk 2 above)
Comment (SivanYeh) [reply layouts/ccip.vue]: Done!

File: pages/about/staff.vue [hunk 4]
```diff
@@ -39,6 +39,13 @@ import CoreH1 from '@/components/core/titles/H1'
 import i18n from '@/i18n/about/staff.i18n'
 
 export default {
+    layout(context) {
+        const ccip = context.query.ccip // to determine if it's opass mobile app
+        if (ccip) {
+            return 'ccip'
+        }
+        return 'default'
```
Comment (yungshenglu) [inline pages/about/staff.vue:47]: Minor: prefer to use `? :`
```
return ccip ? 'ccip' : 'default';
```

**Review by mattwang44 — APPROVED** [review]
LGTM
Please modify the PR based on David's comments.

File: pages/about/staff.vue (hunk 4 above)
Comment (SivanYeh) [reply pages/about/staff.vue:47]: 原來是這樣用!

File: layouts/ccip.vue [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+<style>
+html {
+    @apply bg-black-900 font-sans text-primary-100;
+    font-size: 20px;
+    word-spacing: 1px;
+    -ms-text-size-adjust: 100%;
+    -webkit-text-size-adjust: 100%;
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
+    box-sizing: border-box;
```
[the remark this answers is not in the document]
Comment (SivanYeh) [reply layouts/ccip.vue:31]: 好建議! 不過我的tailwind還不夠純熟. 先維持這樣. 之後再開一個PR改語法

**Review by yungshenglu — APPROVED** [review]
LGTM

**Review by josix — APPROVED** [review]

**Pull request merged**